fastfold/model/fastnn/ops.py

```diff
@@ -318,8 +318,13 @@ def forward(self, in_data, mask, nonbatched_bias=None):
             logits = torch.matmul(q, k.transpose(-1, -2))
 
             if nonbatched_bias is not None:
+                # logits += bias.unsqueeze(1)
+                # logits += (1e9 * (mask_part - 1))[..., :, None, None, :]

```

Comment:
Is it useless now?

Reply:
Just want to keep it as a hint of what function of the kernel.